README.md

```diff
@@ -18,11 +29,73 @@ And then execute:
 
 Or install it yourself as:
 
-    $ gem install structured_logger
+    $ gem install logr
 
 ## Usage
 
-TODO: Write usage instructions here
+An example is worth a thousand words they say, so here it is:
+```ruby
+require 'logr'
+
+class YourClass
+  def self.logger
+    @logger ||= Logr::Logger.new('your-logger-name')
+  end
+
+  .
+  .
+  .
+
+  # A complex event that you want to monitor and has metrics associated
+  YourClass.logger.event('event-name', arbitrary: 'context', add_what: 'you_need')
+                  .monitored('Title of the event', 'A longer description.')
+                  .metric('metric-name', 'add as many as you like')
```

Comment:
I'll add tags to metrics later PR. It's a good idea, but I don't think it's immediately important.